_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -24,20 +24,20 @@ The timing could not be more perfect with the sudden increase in teleworking, wh
 - [Fields with domains currently only display coded values](https://github.com/agrc/open-sgid/issues/7)
 
-#### Differences from SQL SDE database
+#### Differences from SQL SDE Database
 
 - [Table names and casing](https://github.com/agrc/open-sgid/issues/12)
 - [Attribute names and casing](https://github.com/agrc/open-sgid/issues/13)
 - [Null value sorting behavior](https://github.com/agrc/open-sgid/issues/6)
 - [ArcMap requires selection of unique ID field](https://github.com/agrc/open-sgid/issues/9)
 
-Tags are assigned to each issue to provide a level of categorization. This repository will be an important reference for Open SGID users to search for information about known issues, and to submit new issues they encounter, so that AGRC staff and the general user population can collectively offer feedback and solutions. Use the green New issue button on the [issues page](https://github.com/agrc/open-sgid/issues) to submit yours, and visit this page when you first come across a problem with, or question about, the Open SGID to see if others have already found a solution.
+Tags are assigned to each issue to provide a level of categorization. This repository will be an important reference for Open SGID users to search for information about known issues, and to submit new issues they encounter, so AGRC staff and the general user population can collectively offer feedback and solutions. Use the green New issue button on the [issues page](https://github.com/agrc/open-sgid/issues) to submit yours, and visit this page when you first come across a problem with, or question about, the Open SGID to see if others have already found a solution.
 
-Although it may take some time to adjust to the PostgresSQL database format for traditional Esri clients that are familiar with the SQL SDE database, we view the Open SGID as an enhancement to AGRC’s data access points. It provides a direct SGID database connection that is no longer constrained to the Utah state government network, thereby living up to its name as a database that is truly open and accessible to all GIS platforms and users. We look forward to your feedback as both first-time and long-time SGID users utilize Utah’s most comprehensive and authoritative, open source, geospatial database for your cartographic and analytical workflows.
+Although it may take some time to adjust to the PostgresSQL database format for traditional Esri clients that are familiar with the SQL SDE database, we view the Open SGID as an enhancement to AGRC’s data access points. It provides a direct SGID database connection that is no longer constrained to the Utah state government network, thereby living up to its name as a database that is truly open and accessible to all GIS platforms and users. We look forward to your feedback as both first-time and long-time SGID users utilize Utah’s most comprehensive and authoritative, open-source, geospatial database for your cartographic and analytical workflows.
```

Comment:
```suggestion
Although it may take some time for traditional Esri clients who are familiar with the SQL SDE database to adjust to the PostgresSQL database format, we view the Open SGID as an enhancement to AGRC’s data access points. It provides a direct SGID database connection that is no longer constrained to the Utah state government network, thereby living up to its name as a database that is truly open and accessible to all GIS platforms and users. We look forward to your feedback as both first-time and long-time SGID users utilize Utah’s most comprehensive and authoritative open-source geospatial database for your cartographic and analytical workflows.
```